packages/now-next/src/index.ts

```diff
@@ -84,6 +85,8 @@ interface BuildParamsType extends BuildOptions {
 }
 
 export const version = 2;
+// Limit for max size each lambda can be, currently 45 MB
+const lambdaCompressedByteLimit = 45 * 1000 * 1000;
```

Comment:
I thought the limit was was 50MB?

Reply:
Yeah initially left a little room since I wasn't including the compressed size of the page but this is factored in now so bumped it back to `50MB` and also updated to honor the `maxLambdaSize` config if set to allow benchmarking different lambda sizes easier